gammapy/utils/testing.py

```diff
@@ -1,20 +1,25 @@
 # Licensed under a 3-clause BSD style license - see LICENSE.rst
 """Utilities for testing"""
 from __future__ import absolute_import, division, print_function, unicode_literals
+
 import os
+
 from astropy.tests.helper import pytest
-from astropy.utils.data import get_pkg_data_filename
+from astropy.utils import minversion
 from ..data import DataManager
 
 __all__ = [
     'requires_dependency',
     'requires_data',
+    'SHERPA_LT_4_8'
 ]
 
+# Todo : Change to 4_8 once there is a sherpa release
```

Comment:
I did not know that there was a sherpa 4.8 release. The head version I was using was ``4.7+642`` and I wanted the tests to run locally.